dockerfiles/pyne_dag_openmc

```diff
@@ -0,0 +1,197 @@
+FROM debian:bookworm-slim
+
+# modified from OpenMC's dockerfile https://github.com/openmc-dev/openmc/blob/develop/Dockerfile
+# Install and update dependencies from Debian package manager
+RUN apt-get update -y && \
+    apt-get upgrade -y && \
+    apt-get install -y \
+        python3-pip \
+        python-is-python3 \
+        wget \
+        git \ 
+        build-essential \
+        cmake \
+        mpich \
+        libmpich-dev \
+        libpng-dev \
+        liblzma-dev \
+        make \
+        build-essential \
+        libssl-dev \
+        zlib1g-dev \
+        libbz2-dev \
+        libreadline-dev \
+        libsqlite3-dev \
+        wget \
+        ca-certificates \
+        curl \
+        llvm \ 
+        libncurses5-dev \
+        xz-utils \
+        tk-dev \
+        libxml2-dev \
+        libxmlsec1-dev \
+        libffi-dev \
+        liblzma-dev \
+        mecab-ipadic-utf8 \
+        git \
+        libeigen3-dev \
+        libnetcdf-dev \
+        libtbb-dev \
+        libglfw3-dev \
+        libblas-dev \
+        liblapack-dev \
+        wget \
+        nano
+
+ENV HOME=/root
+
+ENV PYTHON_VERSION 3.11.9
+
+# Set-up necessary Env vars for PyEnv
+ENV PYENV_ROOT /root/.pyenv
+ENV PATH $PYENV_ROOT/shims:$PYENV_ROOT/bin:$PATH
+
+# Install pyenv & python dependencies.
+RUN set -ex \
+    && curl https://pyenv.run | bash \
+    && pyenv update \
+    && pyenv install $PYTHON_VERSION \
+    && pyenv global $PYTHON_VERSION \
+    && pyenv rehash \
+    && pip install --upgrade pip \
+    && pip install packaging \
+    progress pyyaml pytest openmc-plasma-source vtk tables
+
+ARG compile_cores=18
+
+# HDF5 stuff
+ENV HDF5_BUILD_DIR=/root/build/hdf5
+ENV HDF5_INSTALL_DIR=/opt/hdf5
+
+# Embree variables
+ENV EMBREE_TAG='v4.3.1'
+ENV EMBREE_REPO='https://github.com/embree/embree'
+ENV EMBREE_INSTALL_DIR=$HOME/EMBREE/
+
+# MOAB variables
+ENV MOAB_TAG='5.3.0'
+ENV MOAB_REPO='https://bitbucket.org/fathomteam/moab/'
+
+# Double-Down variables
+ENV DD_TAG='v1.1.0'
+ENV DD_REPO='https://github.com/pshriwise/double-down'
+ENV DD_INSTALL_DIR=$HOME/Double_down
+
+# DAGMC variables
+ENV DAGMC_BRANCH='v3.2.3'
+ENV DAGMC_REPO='https://github.com/svalinn/DAGMC'
+ENV DAGMC_INSTALL_DIR=$HOME/DAGMC/
+
```

Comment:
Does it make sense to add a block for OpenMC and PyNE here with relevant version strings, etc?